source/type/data.cpp

```diff
@@ -489,26 +485,54 @@ void Data::aggregate_odt() {
     }
 }
 
-void Data::fill_stop_point_address() {
+void Data::fill_stop_point_address(
+    const std::unordered_map<std::string, navitia::type::Address*>& address_by_address_id) {
     idx_t without_address = 0;
+    bool ntfs_addresses_alowed = false;
+    if (!address_by_address_id.empty()) {
+        ntfs_addresses_alowed = true;
+    }
```

Comment:
To simplify the code, we can use directly if (!address_by_address_id.empty()) instead of declaring a variable non ?

Reply:
this is to avoid to check _empty()_ for all stop points